docs/data/sdks/typescript-react-native/index.md

```diff
@@ -6,13 +6,15 @@ icon: material/react
 
 ![npm version](https://badge.fury.io/js/@amplitude%2Fanalytics-react-native.svg)
 
+The React Native SDK lets you send events to Amplitude. This library is open-source, check it out on [GitHub](https://github.com/amplitude/Amplitude-TypeScript).
+
 !!!beta "Beta SDK Resources"
-    [:material-github: Github](https://github.com/amplitude/Amplitude-TypeScript/tree/main/packages/analytics-react-native) · [:material-code-tags-check: Releases](https://github.com/amplitude/Amplitude-TypeScript/releases) · [:material-book: API Reference](https://amplitude.github.io/Amplitude-TypeScript/)
+    - [React Native SDK Reference :material-book:](https://amplitude.github.io/Amplitude-TypeScript/)
+    - [React Native SDK Repository :material-github:](https://github.com/amplitude/Amplitude-TypeScript/tree/main/packages/analytics-react-native)
+    - [React Native SDK Releases :material-code-tags-check:](https://github.com/amplitude/Amplitude-TypeScript/releases)
 
 --8<-- "includes/no-ampli.md"
```

Comment:
I think it would help to have a direct link to the corresponding SDK with Ampli. Is there a way to parameterize the template to accept a "ampliSdkUrl" to link to the correct language? 

<img width="703" alt="image" src="https://user-images.githubusercontent.com/5790872/185719104-31b79d48-5de4-4d05-bc34-2a4dde7c48f3.png">

e.g. `To use Ampli see the [non-Beta SDK](link) instead.`

Reply:
Added the link in these two pages instead. There are three other pages used no-ampli block but don't have corresponding SDK(unity, flutter, unreal)